app/presenters/activity_presenter.rb

```diff
@@ -0,0 +1,30 @@
+class ActivityPresenter < SimpleDelegator
+  include ActionView::Helpers::SanitizeHelper
+
+  def self.activities(*activities)
+    activities.flatten.map { |activity| ActivityPresenter.new(activity) }
+  end
+
+  def initialize(activity)
+    @activity = activity
+    super
+  end
+
+  def activity_feed_line
```

Comment:
I'd split it to two methods `title` and `type`, so you can have HTML outside using CSS with `type` and the content with `title` (ie don't use HTML inside the presenter unless really needed)

Reply:
```ruby
def title
  "You uploaded #{filename} to #{room_link_for(jam.room)}"
end

def type
  subject_type
end
```
Is this the right idea?